modules/preview-react/text/lib/TokenBased.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import {createComponent} from '@workday/canvas-kit-react/common';
+import {Text, TextProps} from '@workday/canvas-kit-preview-react/text';
```

Comment:
I might be wrong, but I remember when i migrated Box components too layout i accidentally created a circular dependency by importing the component itself in a file like this. Maybe it's just safer to just import from `import {Text, TextProps} from './Text'` ?

Reply:
Maybe there was something different, because my calls don't create any new dependency cycles. Also, I had an issue with docs in Canvas Site when Button wasn't imported from package 🤔 But as we don't get any additional warning, I guess we good with it.